### DISCUSSION TITLE
Page Layout

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#358] [Uncategorized]
Wat is er nodig aan componenten en technieken om zoveel mogelijk Page Layouts te kunnen opmaken.

**Comment by jeffreylauwers** [comment]
Voorstel voor naamgeving van componenten: https://excalidraw.com/

**Comment by jeffreylauwers** [comment]
[Schetsen uitgewerkt in Figma](https://www.figma.com/design/fy08SZpZmqx6ljLwvA3Woe/NLDS---Schetsboek?node-id=4554-6192) op basis van naamgeving zoals getoond in https://excalidraw.com/.

**Comment by RGRMdesign** [comment]
Ik heb laatst voor NORA ook een Page Layout component gemaakt. Zie [nora-page-layout in Figma](https://www.figma.com/design/97dNBzCXJjg3ApxFFrNMlO/Local---NORA---Bibliotheek?node-id=724-796&t=TMxkjZyGJEvSxrUy-1). Onder andere geïnspireerd op het Device Frame component in het Design System van Amsterdam.

Wat ik hieraan ook fijn vindt is om een plek te hebben om een aantal document tokens te documenteren. En hierbij ook aan te geven dat deze in andere componenten overgeërfd kunnen worden. Dit zou in [de opzet](https://excalidraw.com/) van @Robbert dan in het Body component komen, right?

Daarnaast heb ik in dezelfde lijn een Page Body component gemaakt. Zie [nora-page-section in Figma](https://www.figma.com/design/97dNBzCXJjg3ApxFFrNMlO/Local---NORA---Bibliotheek?node-id=732-1088&t=TMxkjZyGJEvSxrUy-1). Ik vond het woord Body wat verwarrend en ben dus voor Section gegaan, wat op zich wel zou kunnen, maar dan moet er wel weer een wrapper omheen zodat je meerdere sections op een pagina kan plaatsen. Zie opzetje in https://excalidraw.com/
